homeassistant/components/coolmaster/climate.py

```diff
@@ -118,6 +100,16 @@ def update(self):
         else:
             self._unit = TEMP_FAHRENHEIT
 
+    @property
+    def device_info(self):
+        """Return device info for this device."""
+        return {
+            "identifiers": {(DOMAIN, self.unique_id)},
+            "name": self.name,
+            "manufacturer": "CoolAutomation",  # TODO: take from `line`
```

Comment:
todos

Reply:
What's the preferred way of documenting those? It required changes in the 3rd party lib, so not likely to happen soon, but still I believe it deserves a mention for future maintenance.

Reply:
Remove them. Person who will update it in the future can figure it out .